cmd/skupper/README.md

```diff
@@ -0,0 +1,41 @@
+# Skupper CLI
+
+The [Skupper](https://skupper.io/) command line enables you to:
+
+* Create a Virtual Application Network (VAN) site
+* Connect to other VAN sites
+* Connect services
+
+## Getting `skupper`
+
+To install `skupper`, download the [latest release](https://github.com/skupperproject/skupper/releases).
+
+
+## Using `skupper`
+
+To create your first VAN site:
+
+```
+skupper init
+skupper connection-token /path/to/mysecret.yaml
+```
+
+To expose a service from the first site:
+
+```
+skupper expose
+```
+
+To connect to the first site from a second site:
+
+```
+skupper connect --secret /path/to/mysecret.yaml
+```
+
+For a complete list of `skupper` commands:
```

Comment:
@ajssmith  is it reasonable to only mention the 'bones' of the commands here, and 'promise' to document them more completely later?

Reply:
That is reasonable to me as we will definitely be iterating/refining over the release horizon.